backend/src/main/java/com/woowacourse/gongcheck/core/application/response/TasksWithSectionResponse.java

```diff
@@ -36,9 +41,9 @@ public static List<TasksWithSectionResponse> from(final Tasks tasks) {
     }
 
     private static TasksWithSectionResponse of(final Section section, final List<Task> tasks) {
-        return new TasksWithSectionResponse(section.getId(), section.getName(),
-                tasks.stream()
-                        .map(TaskResponse::from)
-                        .collect(Collectors.toList()));
+        return new TasksWithSectionResponse(section.getId(), section.getName(), section.getImageUrl(),
+                section.getDescription(), tasks.stream()
+                .map(TaskResponse::from)
+                .collect(Collectors.toList()));
```

Comment:
```suggestion
        return new TasksWithSectionResponse(
                section.getId(),
                section.getName(),
                section.getImageUrl(),
                section.getDescription(),
                tasks.stream()
                        .map(TaskResponse::from)
                        .collect(Collectors.toList())
        );
```
가독성적인 면에서 이렇게 보면 더 가독성이 좋을 수 있을 것 같네요 :)

고려만해주세요~